Source/JavaScriptCore/dfg/DFGNode.h

```diff
@@ -1486,6 +1486,12 @@ struct Node {
     {
         return speculationFromJSType(queriedType());
     }
+
+    uint32_t structureFlags()
+    {
+        ASSERT(op() == HasStructureWithFlags);
+        return m_opInfo.as<uint32_t>();
+    }
```

Comment:
You should add `bool hasStructureFlags() const` method too. And you should add flag dump support in DFGGraph.